### PR TITLE
Drag handles for chapter and section outline items

### PR DESCRIPTION
**Pull request by tf**
Possibility to reorder chapters and items is otherwise hard to
discover.

REDMINE-17873

### CODE REVIEW

**Comment by tf** [comment]
These arrow indicators now show up when the corresponding item is hovered. For sections the arrows respect the invert colors setting of the page to ensure contrast. 

Bildschirmfoto von 2020-08-20 10-45-39
Bildschirmfoto von 2020-08-20 10-45-13

**Comment by aviav** [comment]
**question (non-blocking):** I think there is very little padding to the right of the arrow indicators in the first screenshot. Still, in case you have already though about this or taken this idea from some example, I'd say it's good enough. Did you already think about this padding?

**Comment by tf** [comment]
You mean the padding between the arrows and the preview content of the section? Yeah, it bugged me as well, but the margin of the content inside the preview depends on so many different factors that it did not seem worth it to optimize this one special case. The arrows are only visible on hover, so I thought it might just be good enough for now.

**Review by aviav — APPROVED** [review]

File: entry_types/scrolled/package/src/editor/views/ChapterItemView.module.css [hunk 1]
```diff
@@ -23,6 +23,15 @@
   color: indicatorIconColor;
 }
 
+.dragHandle {
+  composes: dragHandle from './outline.module.css';
+  color: #444;
```
[the remark this answers is not in the document]
Comment (tf) [reply entry_types/scrolled/package/src/editor/views/ChapterItemView.module.css:28]: Yeah, we do this in some places., but not consistently yet. I'll try to do it more in the future.

**Pull request merged**